.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -0,0 +1,66 @@
+---
+name: bug report
+about: use this template to report bugs
+title: ":bug: [BUG SUMMARY]"
+labels: bug, needs triage
+assignees: MiczFlor
+---
+
+## Hardware
+
+### RaspberryPi version
+
+i.e. `3 B+`
+
+### RFID Reader
+
```

Comment:
Comment how the version can be obtained would be helpful.

Reply:
good idea.